<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CR5DocumentsFilter</class>
 <widget class="QWidget" name="CR5DocumentsFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>375</y>
     <width>158</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*Only for documents in the store</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>89</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Document state</string>
   </property>
  </widget>
  <widget class="C5LineEdit" name="lePaidName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>193</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="C5DateEdit" name="deStart">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>date1</string>
   </property>
  </widget>
  <widget class="C5LineEditWithSelector" name="leDocState">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>89</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>docstate</string>
   </property>
  </widget>
  <widget class="C5DateEdit" name="deEnd">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>37</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>date2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>37</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date end</string>
   </property>
  </widget>
  <widget class="C5LineEdit" name="lePartnerName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>115</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="C5LineEdit" name="leOperatorName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>141</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>63</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Document type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>141</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operator</string>
   </property>
  </widget>
  <widget class="C5LineEdit" name="leDocTypeName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>63</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="C5LineEdit" name="leReasonName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>219</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="C5LineEditWithSelector" name="lePaid">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>193</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>paid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>219</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reason*</string>
   </property>
  </widget>
  <widget class="C5LineEditWithSelector" name="leReason">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>219</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>reason</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date start</string>
   </property>
  </widget>
  <widget class="C5LineEditWithSelector" name="leDocType">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>63</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>doctype</string>
   </property>
  </widget>
  <widget class="C5LineEdit" name="leDocStateName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>89</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>115</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier</string>
   </property>
  </widget>
  <widget class="C5LineEditWithSelector" name="lePartner">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>115</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>supplier</string>
   </property>
  </widget>
  <widget class="C5LineEditWithSelector" name="leOperator">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>141</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>operator</string>
   </property>
  </widget>
  <widget class="C5LineEditWithSelector" name="lePayment">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>167</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="FilterName" stdset="0">
    <string>payment</string>
   </property>
  </widget>
  <widget class="C5LineEdit" name="lePaymentName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>167</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>167</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>193</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Paid</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnFixDate">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>9</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/pushpin.png</normaloff>:/pushpin.png</iconset>
   </property>
   <property name="FixDate" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>deStart</tabstop>
  <tabstop>deEnd</tabstop>
  <tabstop>leDocType</tabstop>
  <tabstop>leDocState</tabstop>
  <tabstop>lePartner</tabstop>
  <tabstop>leOperator</tabstop>
  <tabstop>lePayment</tabstop>
  <tabstop>lePaid</tabstop>
  <tabstop>leReason</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
